src/components/GrayBG.js

```diff
@@ -19,79 +28,52 @@ export default styled.section`
   margin-bottom: 0;
 
   ${breakpoint('smallTablet')`
-    margin-top: ${props =>
-      props.offsetBottom
-        ? 0
-        : remcalc((props.topOffset && props.topOffset * -1) || 50)};
-    padding-top: ${props =>
-      props.offsetBottom ? remcalc(props.topOffset * -1) : 0};
-    margin-bottom: ${props => remcalc(props.topOffset || -50)};
+    margin-top: ${({ topOffset }) =>
+      remcalc((topOffset && topOffset * -1) || PADDING)};
+    margin-bottom: ${({ topOffset }) => remcalc(topOffset || -PADDING)};
     &:after {
-      content: ' ';
-      width: 100%;
-      height: ${props => remcalc(props.topOffset ? props.topOffset * -1 : 50)};
-      bottom: 0;
-      background-color: white;
-      display: block;
-      position: absolute;
-      z-index: 2;
+      ${afterStyles}
+      height: ${({ topOffset }) =>
+        remcalc(topOffset ? topOffset * -1 : PADDING)};
     }
   `}
 
   ${breakpoint('tablet')`
-    margin-top: ${props =>
-      props.offsetBottom
-        ? 0
-        : remcalc((props.topOffset && props.topOffset * -2) || 50)};
-    padding-top: ${props =>
-      props.offsetBottom ? remcalc(props.topOffset * -2) : 0};
-    margin-bottom: ${props => remcalc(props.topOffset * 2 || -50)};
+    margin-top: ${({ topOffset }) =>
+      remcalc((topOffset && topOffset * -2) || PADDING)};
+    margin-bottom: ${({ topOffset }) => remcalc(topOffset * 2 || -PADDING)};
```

Comment:
In the default version (if the topOffset prop is not provided), this seems to be causing too little spacing between GrayBG sections and the section below it:

![image](https://user-images.githubusercontent.com/15656538/50636712-1e7f4100-0f4f-11e9-8340-c4d29ef2f299.png)

Reply:
That is supposed to happen, this is nothing but a bg. You need to then set the padding yourself on your other sections

Reply:
In this particular example, the "From the blog" section _does_ have the appropriate padding attached:
![image](https://user-images.githubusercontent.com/15656538/50696068-5f9e5080-1036-11e9-8115-83d9e1f740be.png)

but because you're using `margin: -50` on the grey component just above it, this is having a greater effect than the padding on the white component.

The pictures below illustrate that it is the negative margin values on the grey component which are causing the incorrect spacing between the grey section and the title in the white section:

### With PADDING=50 on grey section
![image](https://user-images.githubusercontent.com/15656538/50696170-9d02de00-1036-11e9-8d62-7c0e792bc0bc.png)

### Without PADDING=50 on grey section
![image](https://user-images.githubusercontent.com/15656538/50696195-ae4bea80-1036-11e9-9c58-a44d0a78d4dd.png)